pkg/component/etcd/statefulset/statefulset.go

```diff
@@ -961,9 +980,16 @@ func getReadinessHandlerForSingleNode(val Values) corev1.ProbeHandler {
 }
 
 func getReadinessHandlerForMultiNode(val Values) corev1.ProbeHandler {
+	scheme := corev1.URISchemeHTTPS
+	if val.BackupTLS == nil {
+		scheme = corev1.URISchemeHTTP
+	}
+
 	return corev1.ProbeHandler{
-		Exec: &corev1.ExecAction{
-			Command: val.ReadinessProbeCommand,
+		HTTPGet: &corev1.HTTPGetAction{
+			Path:   "/readyz",
+			Port:   intstr.FromInt(int(pointer.Int32Deref(pointer.Int32(9095), *pointer.Int32(9095)))),
+			Scheme: scheme,
```

Comment:
Can we get port values from `Values` fields as similar to `getReadinessHandlerForSingleNode`
And remove field `val.ReadinessProbeCommand`